public/translation_fr.json

```diff
@@ -14,19 +14,19 @@
         "level": "NIVEAU",
         "downtime": "TEMPS LIBRE",
         "status": "VISIBILITÉ",
-        "delete-message": "Êtes-vous certian de vouloir supprimer ce/ces personnage(s)",
+        "delete-message": "Êtes-vous certain de vouloir supprimer ce/ces personnage(s)?",
         "edit-character": "Modifier le personnage",
-        "edit-entry": "Modifier le saisie",
-        "fill-out-fields": "Veuillez remplier les champs suivants avec les détails de votre personnage",
+        "edit-entry": "Modifier la session",
+        "fill-out-fields": "Veuillez remplier les champs suivants avec les détails de votre personnage.",
         "public-status": "Acceptez-vous que ce personnage soit rendu publique?",
         "import-help-text": "Importez les saisies de vos personnages en téléchargeant les logs de ",
         "adventure-league-dot-com": "AdventurersLeagueLog.com ",
```

Comment:
Is there a reason there is a space at the end here?

Reply:
yeah, this is due to formatting, it needs a space at the end or a space in the code, i just put the space here instead